uvtrick/__init__.py

```diff
@@ -35,6 +34,9 @@ def uvtrick_(path, func, *args, **kwargs):
 
     with tempfile.TemporaryDirectory() as temp_dir:
         temp_dir = Path(temp_dir)
+        script = temp_dir / "pytemp.py"
+        output = temp_dir / "tmp.pickle"
```

Comment:
nitpick, and might patch this later myself. 

```suggestion
        script_path = temp_dir / "pytemp.py"
        output_path = temp_dir / "tmp.pickle"
```

The type information is known, so the editor can help ... but I might prefer to have path in the variable name as well.